portal/src/main/scala/cromwell/pipeline/service/GitClient.scala

```diff
@@ -0,0 +1,12 @@
+package cromwell.pipeline.service
+
+import java.io.File
+
+import cromwell.pipeline.datastorage.dto.Project
+
+trait GitClient {
+  def updateFile(project: Project, file: File)
+  def updateListOfFiles(project: Project, files: List[File])
+  def createRepository(project: Project)
+  def getListOfFiles(project: Project, file: File)
```

Comment:
What is this method supposed to do? Why does it require a `File`?

Reply:
@myazinn 
updateFile is supposed to update a file in user's repository on Gitlab,
getListOfFiles is supposed to return all the files from user's Gitlab project. Probably File won't be java.io.File, maybe it would be better to use a case class. Of that I'm not sure it's just a concept so far.